apps/dsaparam.c

```diff
@@ -128,6 +130,11 @@ int dsaparam_main(int argc, char **argv)
         goto end;
 
     if (numbits > 0) {
+        if (numbits > DSA_RECOMMENDED_MAX_KEY_LENGTH) 
+            BIO_printf(bio_err, "# Warning: It is not recommended to use more than %d bit for DSA keys.       #\n"
```

Comment:
Indentation: Insert linebreak before argument 2 (the literal string) and align the two literal strings with the `b` of `bio_err`.